src/components/LanguageSelector/index.tsx

```diff
@@ -95,7 +91,7 @@ const LanguageSelector: React.FC<LanguageSelectorProps> = ({
   });
 
   return (
-    <Select
+    <Select<OptionType, true>
```

Comment:
I think this is the first time I've seen inline generics like this. Wild.

Reply:
Same for me, had to take a little time to look at their type definitions because it's pretty wild in there, but also pretty cool
Here's the typings for the `Select` component:

```ts
export class StateManager<
    OptionType extends OptionTypeBase = { label: string; value: string },
    IsMulti extends boolean = false,
    GroupType extends GroupTypeBase<OptionType> = GroupTypeBase<OptionType>,
    T extends SelectBase<OptionType, IsMulti> = SelectBase<OptionType, IsMulti>
> extends Component<
    StateProps<SelectProps<OptionType, IsMulti, GroupType>> &
        Props<OptionType, IsMulti, GroupType> &
        SelectProps<OptionType, IsMulti, GroupType>,
    State<OptionType, IsMulti>
>
```